### PR TITLE
Feature/property selector

### PR DESCRIPTION
**Pull request by swift-student**
## What's here (& why)

- A new PropertySelectionController which wraps a given view controller and adds a PropertySelector above it
- Some tweaks to the user defaults extension in the Property model
- Moved pull to refresh out of package and directly into project in order to tweak refresh control location
- Some file organization (sorry, couldn't help myself)

I apologize for having so many files changed...

[Trello](https://trello.com/c/PimA55Me/75-property-selection)

### CODE REVIEW

**Review by DeVitoC — APPROVED** [review]
Looks good to me

File: EcoSoapBank/Custom Views/PullToRefresh.swift [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+//
+//  Created by Shawn Gee on 9/22/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import Introspect
+import SwiftUI
+
+private struct PullToRefresh: UIViewRepresentable {
```
Comment (jonbash) [inline EcoSoapBank/Custom Views/PullToRefresh.swift:13]: Hahaha nice

File: EcoSoapBank/Features/Payment/CheckoutViewController.swift [hunk 2]
```diff
@@ -0,0 +1,150 @@
+//
+//  CheckoutViewController.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-28.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
```
Comment (jonbash) [inline EcoSoapBank/Features/Payment/CheckoutViewController.swift:7]: This file can be deleted; it's been replaced with the MakePaymentVC. Must have snuck back in during a merge.

**Review by jonbash — APPROVED** [review]
👌

File: EcoSoapBank/Features/Payment/CheckoutViewController.swift (hunk 2 above)
Comment (swift-student) [reply EcoSoapBank/Features/Payment/CheckoutViewController.swift:7]: Ok, thanks for the heads-up!

**Pull request merged**